ostd/src/arch/x86/tdx_guest.rs

```diff
@@ -25,16 +25,21 @@ pub enum PageConvertError {
     TdVmcall,
 }
 
-/// Sets the given physical address range to Intel TDX shared pages.
-/// Clears the data within the given address range.
-/// Make sure the provided physical address is page size aligned.
+/// Converts physical pages to Intel TDX shared pages.
+///
+/// This function sets the [`PrivFlags::SHARED`] bit in the linear mapping of physical pages. Then,
+/// it invokes TDVMCALLs to convert those pages into Intel TDX private pages. Due to the
```

Comment:
TDVMCALLs -> the `map_gpa` TDVMCALL
into Intel TDX private pages. -> into Intel TDX shared pages.